src/main/java/com/annapolisworks/OpNode.java

```diff
@@ -0,0 +1,27 @@
+package com.annapolisworks;
+
+public class OpNode {
+
+    public OpNode lastNode;
+    public OpNode nextNode;
+    public Operator operator;
+    public double lastNumber;
+    public double nextNumber;
```

Comment:
I think they need to be public. When a node is being simplified, it has to reach into the nodes around it and update some of these with the new number. I could make them private and control this with methods, but is that necessary?